ngraph_bridge/ngraph_builder.cc

```diff
@@ -4915,7 +4915,7 @@ Status Builder::TranslateGraph(
   // ought to be `const Node*`, but GetReversePostOrder doesn't use `const`
 
   vector<Node*> ordered;
-  GetReversePostOrder(*input_graph, &ordered);
+  GetReversePostOrder(*input_graph, &ordered, NodeComparatorName());
```

Comment:
Add a unit test to validate this.

Reply:
`GetReversePostOrder` is a TF function. It stable comparator implementation and testing are in the TF repo. eg, here: https://github.com/tensorflow/tensorflow/blob/c42d974e7abb7380722a6f383084a84d8f3b75b0/tensorflow/core/graph/algorithm_test.cc#L155

Therefore I did not add any unit tests.